### DISCUSSION TITLE
Yalan söyleyen üyeler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#244] [Zeka Sorusu] [answered]
Sonsuzus üyeleri Pazartesi, Salı ve Çarşamba günleri yalan, diğer günler ise doğruyu söylemektedir. Bu üyelerden biri aşağıdaki iki önermeyi yapar:
"Ben dün yalan söyledim"
"Üç gün sonra yine yalan söyleyeceğim"
Önermeleri yaptığı günü bulunuz.

**Comment by Furkanyildirm** [comment, marked as answer]
pazartesi çünkü pazartesi yalan söyleyeceği için "ben dün yalan söyledim" lafı da "Üç gün sonra yine yalan söyleyeceğim" lafı da yalan oluyor ve doğrulanıyor
